app/routes.js

```diff
@@ -3,4 +3,5 @@ const logger = require('./logger'),
 
 exports.init = app => {
   app.post('/users', [], userFunctions.newUser);
```

Comment:
Remove empty array in middlewares `app.method('/something', controller.function)`